.github/workflows/ci.yml

```
@@ -21,6 +21,18 @@ jobs:
       with:
         go-version: '1.22'
 
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        version: "27.2"
+
+    - name: Install protoc-gen-go
+      run: go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0
+
```

Add another step to generate Go language bindings of `compositor`
```suggestion
- name: Install protoc-gen-go-grpc
  run: google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.5.1
```